frontend/next.config.js

```diff
@@ -47,13 +47,43 @@ const moduleExports = {
       ];
    },
    async redirects() {
-      return [
+      const redirects =[
          {
             source: '/Store/Guide/:guideid',
             destination: `${process.env.NEXT_PUBLIC_IFIXIT_ORIGIN}/Guide/_/:guideid`,
             permanent: true,
          },
       ];
+
+      // Do not hide /Parts and /Tools in dev
+      if (process.env.NODE_ENV == "development") {
```

Comment:
I believe the cypress tests are failing because we are running the production build when we are running cypress tests: https://github.com/iFixit/react-commerce/blob/c06691b50e0830eaab40a12938a2e8eb55de819c/.github/workflows/main.yml#L29-L30

For a quick/short-term fix, I think we could run tests on the development build to get cypress working, But ideally I think we want to use the production build for tests?
```diff
- build: pnpm run build 
- start: pnpm start 
+ start: pnpm dev
```